client/src/main/scala/io/delta/sharing/client/util/ConfUtils.scala

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package io.delta.sharing.spark.util
+package io.delta.sharing.client.util
```

Comment:
This seems odd. These are all spark confs. Why should they belong int the client?

Reply:
ConfUtils  is actually used by DeltaSharingFileSystem, and used to initialize DeltaSharingRestClient in RemoteDeltaLog, I'll refactor that later as well.

Reply:
Having DeltaSharingFileSystem in the client seems a bit odd.
Why do we need it in the client?

Reply:
It was discussed in the design doc actually. 
delta-sharing-client serves two main purposes: 1) DeltaSharingClient to handle http requests/responses 2) DeltaSharingFileSystem to handle reading data out of pre-signed urls.